core/trino-server-rpm/src/main/resources/dist/config/env.sh

```diff
@@ -0,0 +1,3 @@
+#JAVA_HOME=
+# Secret configuration properties exported as environment variables
```

Comment:
This is secrets-agnostic, so remove "Secret" from here

Reply:
Done.